charts/karmada/templates/karmada-aggregated-apiserver.yaml

```diff
@@ -29,6 +29,8 @@ spec:
     spec:
       {{- include "karmada.aggregatedApiServer.imagePullSecrets" . | nindent 6 }}
       automountServiceAccountToken: false
+      initContainers:
```

Comment:
`karmada-aggregated-apiserver` should wait for `karmada-apiserver`?

Or we can skip adding the initContainer, because the issues it causes are not significant.

Reply:
`agg-apiserver` really does need to wait for `apiserver`, otherwise it will be failed with following errors:

```console
Defaulted container "karmada-53dujqro77-aggregated-apiserver" out of: karmada-53dujqro77-aggregated-apiserver, wait (init)
I0603 03:38:38.500024       1 options.go:97] karmada-aggregated-apiserver version: version.Info{GitVersion:"v1.10.0-preview4-179-g9c85fbddf", GitCommit:"9c85fbddf914f4af7a05ad7611e12e2e73ffdea1", GitTreeState:"clean", BuildDate:"2024-05-30T03:23:07Z", GoVersion:"go1.21.10", Compiler:"gc", Platform:"linux/amd64"}
E0603 03:38:39.916483       1 run.go:74] "command failed" err="unable to load configmap based request-header-client-ca-file: Get \"https://karmada-53dujqro77-apiserver.karmada-53dujqro77.svc.cluster.local:5443/api/v1/namespaces/kube-system/configmaps/extension-apiserver-authentication\": dial tcp 100.12.20.194:5443: connect: connection refused"
```